src/schedule.cpp

```diff
@@ -31,8 +29,7 @@ const StructuredSDFG& Schedule::sdfg() const { return this->builder_.subject();
 
 analysis::AnalysisManager& Schedule::analysis_manager() { return this->analysis_manager_; };
 
-LoopSchedule Schedule::loop_schedule(
-    const structured_control_flow::ControlFlowNode* loop) const {
+LoopSchedule Schedule::loop_schedule(const structured_control_flow::ControlFlowNode* loop) const {
```

Comment:
Yeah, would be nice to have a parent class in another PR for schedulable CFG elements